### DISCUSSION TITLE
Repo cleanup

### DISCUSSION THREAD

**Discussion by ErinWeisbart** [CellProfiler/CellProfiler-plugins#201] [General]
Final few questions for cleanup of repo:

@callum-jpg We can remove the requirements.txt and requirements-windows.txt from the repo now, right?

@bethac07 We're not using Travis CI anymore so we can remove the .travis.yml, right?

@hinerm Can you clarify what is in the [cpij](https://github.com/CellProfiler/CellProfiler-plugins/tree/master/cpij) folder? Is it necessary? Since we have moved runimagjscript.py to a new folder (`active_plugins` along with all other active plugins), does it need to move too?

**Comment by bethac07** [comment]
Yes, Travis can be removed; cpij is needed and should move alongside the plugin.

**Comment by callum-jpg** [comment]
Yep, anything that was contained in the requirement files will be kept in setup.py in future
